Windows eslint failed

Following is the output of the failed job

```shell
No files matching the pattern "'--ext'" were found.
Please check for typing mistakes in the pattern.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Process completed with exit code 1.
```

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,2 @@
+# Ensuring github workflows eol for all platform is lf
+* eol=lf

--- a/package.json
+++ b/package.json
@@ -14,8 +14,8 @@
     "start": "tsdx watch",
     "build": "tsdx build && node ./scripts/flowgen.js",
     "test": "tsdx test",
-    "lint": "eslint '--ext' '.js,.ts' './src' './test'",
-    "lint-fix": "eslint '--ext' '.js,.ts' './src' './test' --fix",
+    "lint": "eslint ./src ./test --ext .js,.ts",
+    "lint-fix": "eslint ./src ./test --ext .js,.ts --fix",
     "prepare": "tsdx build",
     "size": "size-limit",
     "analyze": "size-limit --why"

